code/game/objects/items/devices/flashlight.dm

```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] eyes and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
I'm saying if he's going to use "the" it needs to be "\the" so it doesn't come out as "the the"